srcpkgs/python3-txtorcon/template

```diff
@@ -1,17 +1,19 @@
 # Template file for 'python3-txtorcon'
 pkgname=python3-txtorcon
-version=20.0.0
-revision=2
+version=21.1.0
+revision=1
 wrksrc="txtorcon-${version}"
 build_style=python3-module
 hostmakedepends="python3 python3-setuptools"
 depends="python3-automat python3-Twisted"
```

Comment:
I think we are missing some dependencies, could you verify them? - six, zope.interface and cryptography

Reply:
these should be dependencies of python3-automat, python3-Twisted, and python3-openssl (from Twisted)

Reply:
It seems like they all are pulled in by other dependencies, but I would list them explicitly anyway, because txtorcon imports them.

Reply:
Do you think this would be a good idea to add to `Manual.md` (ie: add dependencies if the package imports them)?